### PR TITLE
add emoji-recall

### PR DESCRIPTION
**Pull request by lujun9972**

### CODE REVIEW

File: recipes/emoji-recall [hunk 1]
```diff
@@ -0,0 +1 @@
+(emoji-recall :fetcher github :repo "lujun9972/emoji-recall.el" :files ("emoji-recall.el" "emoji-cheat-sheet/*.png"))
```
Comment (milkypostman) [inline recipes/emoji-recall]: please change `"emoji-recall.el"` to `:defaults`

**Comment by milkypostman** [comment]
i think that only `emoji-recall-game-start` needs to be autoloaded.

if you are going to use lexical-binding (do you really need to?) then you need to add a `package-requires: ...` for emacs 24.

i *think* the format is, `Package-Requires: ((emacs 24))`

**Comment by vyp** [comment]
@milkypostman I think 24 needs to be in double quotes, but yes.

**Comment by milkypostman** [comment]
thank you, my review skills are rusty. or nonexistent. one of those

**Comment by lujun9972** [comment]
Yes, we do need lexical-binding. so I have added the package requires and changed "emoji-recall.el" to :defaults

**Pull request merged**